src/regedit/RegEdit.jl

```diff
@@ -0,0 +1,11 @@
+module RegEdit
```

Comment:
Internal extra modules like this is usually just more annoying than helpful IMO.

Reply:
In the end the rest of the code should be calling exactly one function in here (`register`). Having it in a module will help prevent leaky abstractions, and will enable branching it off into a separate package (which should probably happen eventually, given how different the functionality is between this and the server).

Reply:
I've definitely run into a few cases where it's annoying but I believe this will not be one of them.